lib/spack/spack/test/llnl/util/cpu.py

```diff
@@ -207,7 +207,8 @@ def test_optimization_flags(target_name, compiler, version, expected_flags):
 
 
 @pytest.mark.parametrize('target_name,compiler,version', [
-    ('excavator', 'gcc', '4.8.5')
+    ('excavator', 'gcc', '4.8.5'),
+    ('skylake', 'gcc', 'string')
```

Comment:
use a more descriptive name than `string` here (like "cannot_build_for_skylake" with a comment indicating it's testing string versions or something)